testcases/GapTestCase.py

```diff
@@ -83,6 +84,10 @@ def test_btp_GAP_CONN_DCON_1(self):
         The IUT1 is operating in the Peripheral role.
         """
 
+        if self.iut1.get_type() == self.iut1.TYPE_ANDROID or \
```

Comment:
I wonder if we should have had something like `issues.yml` file with entries like `test_name: issue`. And if test has known issues it's not being executed. What you have here is OK imo, just a loose idea. Of course this should also be IUT dependent

Reply:
I was so pumped with the idea that I forgot you mentioned yml and I made a json file, that shouldn't be an issue but if it is please let me know!

I also added the reason for the skip.

Thanks!

Reply:
I have nothing agains json, looks good to me :)

Reply:
I see that every tests calls `verify_skipped`. Maybe it would be possible to move this check to `create_suite` level, and just skip adding it to suite?